src/Entity/RevokableTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-authentication-oauth2 for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-authentication-oauth2/blob/master/LICENSE.md
+ *     New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Authentication\OAuth2\Entity;
+
+trait RevokableTrait
+{
+    /**
+     * @var bool
+     */
+    protected $revoked;
```

Comment:
`private`

Reply:
`protected` has been preferred in order to remain consistent with [League's traits implementation](https://github.com/thephpleague/oauth2-server/tree/master/src/Entities/Traits). 
Furthermore, I don't see why inheritance of those properties would not be allowed.

Reply:
I think consistency with the upstream implementation makes sense here.